static/js/base.js

```diff
@@ -155,6 +158,11 @@ function updateSensors(sensorValues) {
     document.getElementById("right-sensor").innerText = "Right Sensor: " + sensorValues.rightSensor.toFixed(3);
 }
 
+function updateSwitches(switchValues) {
+    document.getElementById("front-switch").innerText = "Front Switch: " + switchValues.frontSwitch; 
+    document.getElementById("back-switch").innerText = "Back Switch: " + switchValues.backSwitch; 
```

Comment:
The UI should read `True` and `False` (capitalized) rather than `true` and `false`